src/components/Categories.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+const Categories = () => (
+  <div>
+    <h2>Under construction</h2>
```

Comment:
- [ ] Good job so far :+1: But according to the project requirement the Categories should display a button "Check status" only. Please fix it in order to make your project complete and works as expected :+1:

Reply:
Fixed, thank you.